.github/compose/ci.docker-compose.yml

```diff
@@ -34,14 +34,6 @@ services:
     <<: *tests-template
     image: registry.gitlab.com/buildstream/buildstream-docker-images/testsuite-debian:10-${CI_IMAGE_VERSION:-latest}
 
-  ubuntu-18.04:
-    <<: *tests-template
-    image: registry.gitlab.com/buildstream/buildstream-docker-images/testsuite-ubuntu:18.04-${CI_IMAGE_VERSION:-latest}
```

Comment:
while dropping support for python2.6 maybe be ok, I think we still need to check bst-1 runs fine in ubuntu-18.04 and centos-7 (ubuntu has python2.7 for example)

Reply:
Do you mean 3.7? It has indeed both 3.7 and 3.8. But ubuntu 18.04 is pretty old, and we should probably use 20.04 at this point.

Reply:
Yeah sorry I meant 3.7

Reply:
CCing @ssssam as he might have an opinion on this

Reply:
Ah wait, this is for master, nevermind